spec/system/search_spec.rb

```diff
@@ -85,4 +85,26 @@
       expect(search_page).to have_warning_message
     end
   end
+
+  describe "when search menu on desktop" do
+    before do
+      SearchIndexer.enable
+      SearchIndexer.index(topic, force: true)
+      SearchIndexer.index(topic2, force: true)
+    end
+
+    after { SearchIndexer.disable }
+
+    it "soft loads last topic results after navigating away, then back" do
+      visit("/")
+      search_page.click_search_icon
+      search_page.type_in_search_menu("test")
+      search_page.click_search_menu_link
+      expect(search_page).to have_css(".search-menu .results .search-result-topic")
+
+      search_page.find(".topic-list-body tr:first-of-type").click
+      search_page.click_search_icon
+      expect(search_page).to have_css(".search-menu .results .search-result-topic")
```

Comment:
Do we want to be a bit more precise here and ensure the first result here is the same as the one shown after the first search. Maybe we should ensure that the search menu shows `topic.title` and `topic2.title`?